### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by dubeyaditya29**
Added homePage for Cmart.
Screenshot (83)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  </head>
+  <body>
+    <nav class="navbar">
+      <div class="logo">
+        <a href="#" class="logo_link">
+          <i class="fas fa-birthday-cake fa-lg cake-logo"></i>
+        </a>
+        <h1 class="heading-title">Cmart</h1>
+      </div>
+      <ul class="list"></ul>
```
Comment (ttahm3d) [inline index.html:22]: If this element is not doing anything, please remove it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Cmart | Login</title>
```
Comment (junaidshaikh-js) [inline index.html]: I guess the title should be something related to home page, right?

**Review by junaidshaikh-js — APPROVED** [review]
Nice work done! Can be merged.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+          </a>
+        </li>
+        <li class="list_item">
+          <a
+            class="list_item--link"
+            href="https://github.com/dubeyaditya29/Cmart"
+          >
+            <i class="fab fa-github"></i>
+          </a>
+        </li>
```
Comment (ttahm3d) [inline index.html]: These links are there in both header and footer. Since it is a E-commerce app, it would be better to have them in footer.

**Review by ttahm3d — APPROVED** [review]
Have  added few comments please take a look

File: index.html (hunk 2 above)
Comment (dubeyaditya29) [reply index.html]: yes @junaidshaikh-js  forgot to change that thanks for pointing out

File: index.html (hunk 3 above)
Comment (dubeyaditya29) [reply index.html]: Sure thanks for the suggestion.

File: index.html (hunk 1 above)
Comment (dubeyaditya29) [reply index.html:22]: sure @ttahm3d  that is not doing anything removing that. :)

**Comment by dubeyaditya29** [comment]
Hi @junaidshaikh-js  and @ttahm3d  have made the relevant changes.

**Pull request merged**